src/graphql.js

```diff
@@ -47,8 +47,14 @@ exports.assertError = (response, path, messageTest) => {
 exports.assertSuccess = (response) => {
   assert(response.statusCode >= 200 && response.statusCode < 300, `Did not succeed. HTTP status code was ${response.statusCode}`);
 
-  const messages = map(response.body.errors, 'message').join(', ');
-  assert(!response.body.errors, `Did not succeed. Errors were ${messages}`);
+  const errors = map(response.body.errors, err => {
+    return {
+      message: err.message,
+      path: err.path.join('.')
```

Comment:
Maybe I'm being paranoid, but I thought that some errors did not have paths associated with them. For example, if a failure to execute something before the resolves. For example, checkout line 32 above. That hints that sometimes the path does not exist.

Reply:
Oh - good catch! There's even a comment on line 21 that I completely missed: `path isn't defined on schema type errors. Get first error in that case`